release-notes/opensearch-dashboards.release-notes-3.0.0.md

```diff
@@ -0,0 +1,192 @@
+# VERSION 3.0.0 Release Note
+
+### 💥 Breaking Changes
+
+### Deprecations
+
+ - Remove data enhancements config and readonly flag. Removes dead url link, ([#7291](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7291))
+
+### 🛡 Security
+
+ - [CVE-2024-28863] Bump tar from 6.1.11 to 6.2.1 ([#6492](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6492))
```

Comment:
nit: I see there are styles inconsistent 
[CVE-2024-28863] Bump tar from `6.1.11` to `6.2.1`

Reply:
maybe we could define a style like the above and ask LLM to fix the rest and check typos.